packages/integration-tests/suites/replay/errors/droppedError/test.ts

```diff
@@ -17,49 +17,58 @@ import { getReplayEvent, shouldSkipReplayTest, waitForReplayRequest } from '../.
  * TODO: Once we have lifecycle hooks, we should revisit this test and make sure it behaves as expected.
  *       This means that the recording should not be started or stopped if the error that triggered it is not sent.
  */
-sentryTest(
-  '[error-mode] should start recording if an error occurred although the error was dropped',
-  async ({ getLocalTestPath, page }) => {
-    if (shouldSkipReplayTest()) {
-      sentryTest.skip();
-    }
+for (let i = 0; i < 100; i++) {
```

Comment:
we can remove this now?

Reply:
jup, just waiting for the final test run 👍